oauth2/src/main/java/com/okta/spring/boot/oauth/config/OktaOAuth2Properties.java

```diff
@@ -71,6 +71,11 @@
      */
     private String postLogoutRedirectUri;
 
+    /**
+     * Identifies Opaque token
```

Comment:
Make sure this is in proper Javadoc format, and more verbose. As this text is used for the generated complication used in IDEs

Reply:
will do.

Reply:
Updated to:

` /**
     * Signals need for opaque token validation/introspection (if set to true).
     */
`